info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java

```diff
@@ -91,23 +90,23 @@ public static void initialize() throws InterruptedException, IOException {
     datastore = datastoreHelper.getOptions().getService();
     infoCompiler = new InfoCompiler(datastore);
 
-    ELECTION_JSON = new JsonObject();
-    ELECTION_JSON.addProperty("kind", "civicinfo#electionsqueryresponse");
-    JsonArray elections = new JsonArray(1);
     JsonObject election = new JsonObject();
     election.addProperty("id", "2000");
     election.addProperty("name", "VIP Test Election");
     election.addProperty("electionDay", "2013-06-06");
+    JsonArray elections = new JsonArray();
     elections.add(election);
+    ELECTION_JSON = new JsonObject();
```

Comment:
ELECTION_JSON and SINGLE_CONTEST_JSON shouldn't be all-caps, since they're not const (`static final`)